src/modules/Popup/Popup.js

```diff
@@ -250,15 +250,16 @@ export default class Popup extends Component {
       portalProps.closeOnPortalMouseLeave = true
       portalProps.mouseLeaveDelay = 300
     }
-
-    if (on === 'click') {
+    if (_.indexOf(on, 'click') > -1) {
```

Comment:
We can normalize value there and then use it:

```js
const normalizedOn = _.isArray(on) ? on : [on]
```

This will allow to pass `on='hover'` and `on={['hover']}`